pandas/tests/series/test_analytics.py

```diff
@@ -1979,6 +2032,43 @@ def test_map_dict_with_tuple_keys(self):
         tm.assert_series_equal(df['labels'], df['expected_labels'],
                                check_names=False)
 
+    def test_map_box(self):
```

Comment:
I would mov all ``apply,map`` to new file ``tests/series/test_apply_map.py`` (similar to how frame is done)